src/components/atoms/icon.tsx

```diff
@@ -0,0 +1,45 @@
+import { BsPerson } from 'react-icons/bs';
+import { GoHome, GoSearch } from 'react-icons/go';
+import { PiMoon, PiClipboard } from 'react-icons/pi';
+import { IoLanguage } from 'react-icons/io5';
+import {
+  AiOutlineHeart, AiOutlineLike, AiOutlineStar, AiOutlineClose, AiOutlineLeft, AiOutlineBell,
+} from 'react-icons/ai';
+import { HiUserGroup, HiCreditCard } from 'react-icons/hi';
+
+function Icon({ name, size, ariaLabel }:{ name:string; size:string; ariaLabel:string }) {
+  switch (name) {
+    case 'BsPerson': // 사람
```

Comment:
```suggestion
    case 'person': // 사람
```
전달받는 name을 `'BsPerson'` 처럼 react-icons의 아이콘 이름을 그대로 전달받는게 아니라 `'person'` 처럼 일반적인 명사로 바꿔주세요.
지금의 경우에는 추후에 BsPerson 아이콘을 GoPerson 처럼 다른 아이콘으로 변경 시 해당 컴포넌트를 사용한 곳에서 name 까지 전부 바꿔 주어야 하므로 컴포넌트를 만드는 의미가 없을 것 같아요.

Reply:
수정했습니다.